materials/demo.py

```diff
@@ -0,0 +1,39 @@
+"""Demonstration."""
+import numpy as np
+import matplotlib.pyplot as plt
+import material
+
+def main():
+    # pylint: disable=no-member
+    filename = '../materials_data/Al_6061.yaml'
```

Comment:
A recommendation: Use `here = os.path.dirname(os.path.realpath(__file__))` to get the path to this file, and then do path operations on that. That way, this file can be executed while the user is in any directory, and the correct thing will happen

Reply:
fixed by https://github.com/mvernacc/material-properties-interchange/pull/1/commits/91cb4591bf45a176d8d67dcc8065b43204229c2c